ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -37,6 +37,11 @@ export type LegendPositionOptions = {
   position: LegendPosition;
 };
 
+export const DEFAULT_UNIT: UnitOptions = {
```

Comment:
It'd be great to also import this constant in TimeSeriesChartPanel ([see lines here](https://github.com/perses/perses/blob/main/ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx#L44-L47)) so these default properties are always the same since I hard-coded them in that component currently

Reply:
Good catch! I had a note to fix that, and I forgot about it. Will get that now.